src/webrtc/stats/media/mediaTrackStats.ts

```diff
@@ -101,4 +104,28 @@ export class MediaTrackStats {
     public resetBitrate(): void {
         this.bitrate = { download: 0, upload: 0 };
     }
+
+    public set alive(isAlive: boolean) {
+        this.isAlive = isAlive;
+    }
+
+    public get alive(): boolean {
```

Comment:
The explanation of what 'alive' means could go here maybe?

Reply:
Good point. I will add descriptions here.